app/views/posts/partials/_unlike_button.html.erb

```diff
@@ -0,0 +1,3 @@
+<p> unlike button goes here </p>
+<%= @post.inspect %>
+<%= @post.likes.count  %>
```

Comment:
Given this partial is so simple I'd put the conditional here to render either like or unlike, instead of the main view. Moreover, it's better design because otherwise you have to remember to add that `if` on every place you want such a button, but if the condition lived here you'd only have a single way to render the button (and it'd always be right).